src/app/components/Profile/Profile.tsx

```diff
@@ -0,0 +1,39 @@
+import defaultAvatar from './defaultAvatar.svg';
+import classes from './Profile.module.css';
+import { useCurrentUser } from '../../utils/user';
+
+function openLoginDialog() {
+  overwolf.profile.openLoginDialog();
+}
+
+const User = (): JSX.Element => {
+  const [currentUser, profileError] = useCurrentUser();
+
+  return (
+    <section className={classes.container}>
+      <div className={classes.header}>
+        {currentUser?.avatar ? (
```

Comment:
Maybe the svg could be embedded like this `src={currentUser.avatar || defaultAvatar}` as default src in the img tag?

Reply:
the svg is imported as a react component, not as the path itself, so source would not work  via scr